### PR TITLE
XW-1903 | Add dropdown styles

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-1903
https://github.com/Wikia/design-system-reference-page/pull/62

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

File: components/_dropdowns.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+.wds-dropdown-toggle {
+	position: relative;
+	// dropdown content has shadow that should be hidden below the toggle
+	z-index: $dropdown-content-z-index + 1;
+}
+
+.wds-dropdown-content {
+	background-color: $color-white;
+	box-shadow: 0 0 10px 0 rgba(0, 0, 0, .2);
```
Comment (kvas-damian) [inline components/_dropdowns.scss]: I'd feel better if we have have variable for black color ;)

**Comment by kvas-damian** [comment]
One minor comment, besides that it LGTM.

File: components/_dropdowns.scss (hunk 1 above)
Comment (rogatty) [inline components/_dropdowns.scss]: Let's address it later when we decide what to do with `$color-black` which isn't black.

**Pull request merged**